Providers/Profile.swift

```diff
@@ -409,7 +409,13 @@ public class BrowserProfile: Profile {
         return Singleton.instance
     }()
 
-    let accountConfiguration: FirefoxAccountConfiguration = ProductionFirefoxAccountConfiguration()
+    var accountConfiguration: FirefoxAccountConfiguration {
+        let syncService: Bool = self.prefs.boolForKey("accountService") ?? true
```

Comment:
I don't think this should default to `true`. If the pref does not exist (default) then we want to default to regular sync services.